ls1intum/Artemis
# [Bugfix] fix deletion of results

### Checklist

### Motivation and Context


For some reason the endpoints in the client didn't match the endpoints in the server anymore, 
which made deletion of results impossible. This is fixed now.
![image](https://user-images.githubusercontent.com/33342534/123832718-ef6e9e00-d905-11eb-92ad-1cac563d8cdf.png)

### Testing steps
1. course overview -> exercises page
2. for each exercise type click on the submissions button. From this page, navigate to the page for individual submissions. (Click the number in the submissions column)
3. If there is result for the submission it should be visible now, and a 'delete' button should be visible. Using this button should work for all exercise types
![image](https://user-images.githubusercontent.com/33342534/123832659-dcf46480-d905-11eb-9a69-ee5645b378bb.png)

### Screenshots
